moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+  <arg name="limited" default="false"/>
+  <arg name="paused" default="false"/>
+  <arg name="gui" default="true"/>
+  
+  <!-- startup simulated world -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch">
+    <arg name="world_name" default="worlds/empty.world"/>
+    <arg name="paused" value="$(arg paused)"/>
+    <arg name="gui" value="$(arg gui)"/>
+  </include>
+
+  <!-- send robot urdf to param server -->
+  <param unless="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <param if="$(arg limited)" name="robot_description" [URDF_LOAD_ATTRIBUTE] />
+
+  <!-- push robot_description to factory and spawn robot in gazebo -->
+  <node name="spawn_gazebo_model" pkg="gazebo_ros" type="spawn_model" args="-urdf -param robot_description -model robot -z 0.1" respawn="false" output="screen" />
+
+  <include file="$(find [GENERATED_PACKAGE_NAME])/launch/ros_controllers.launch"/>
```

Comment:
Is the _ros_controllers.launch_ file supposed to be generated automatically as well?

Reply:
Yes, ros_controllers.launch can now be generated automatically by the setup assistant.